### PR TITLE
add `ignoreKeywords` option for `value-keyword-case`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://github.com/stylelint/stylelint/issues/1082

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@jeddy3 ping

File: src/rules/value-keyword-case/index.js [hunk 1]
```diff
@@ -50,6 +57,10 @@ export default function (expectation) {
 
         if (parsedUnit !== false) { return }
 
+        const ignoreKeywords = options && options.ignoreKeywords || []
+
+        if (ignoreKeywords.indexOf(keyword) !== -1) { return }
```
Comment (jeddy3) [inline src/rules/value-keyword-case/index.js]: Like [`block-opening-brace-space-before`](https://github.com/stylelint/stylelint/blob/master/src/rules/block-opening-brace-space-before/index.js), can we use `matchesStringOrRegExp` here.

Comment (evilebottnawi) [inline src/rules/value-keyword-case/index.js]: @yep

**Comment by evilebottnawi** [comment]
@jeddy3 ping

**Comment by jeddy3** [comment]
Thanks for updating the docs too!

@davidtheclark This looks good to me. But I won't mind, when, you have a chance casting your eyes over it.

File: src/rules/value-keyword-case/README.md [hunk 2]
```diff
@@ -95,3 +95,131 @@ a {
   transition: -WEBKIT-TRANSFORM 2s;
 }
 ```
+
+## Optional options
+
+### `ignoreKeywords: ["/regex/", "non-regex"]`
+
+Ignore case of keywords values.
```
Comment (davidtheclark) [inline src/rules/value-keyword-case/README.md]: "Ignore the case of specified keywords."

**Comment by davidtheclark** [comment]
Thumbs up.

**Pull request merged**